src/net.cpp

```diff
@@ -2915,10 +2911,8 @@ uint64_t CConnman::CalculateKeyedNetGroup(const CAddress& address) const
     return GetDeterministicRandomizer(RANDOMIZER_ID_NETGROUP).Write(vchNetGroup.data(), vchNetGroup.size()).Finalize();
 }
 
-void CaptureMessageToFile(const CAddress& addr,
-                          const std::string& msg_type,
-                          Span<const unsigned char> data,
-                          bool is_incoming)
+// Dump binary message to file, with timestamp.
+static void CaptureMessageToFile(const CAddress& addr, const std::string& msg_type, Span<const unsigned char> data, bool is_incoming)
```

Comment:
nit: not sure this is an improvement? I find the existing format easier to read, with less LoC change.